src/dehb/utils/config_repository.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+import numpy as np
+
+
+@dataclass
+class ConfigItem:
+    config_id: int
+    config: np.array
+    fidelities: dict
```

Comment:
dict of what?
maybe we write a comment that we only support discrete fidelities?

Reply:
Should be `dict[float, ResultItem]`.
However we do support continuous fidelities AFAIK